lib/rspec/rails/matchers/active_job.rb

```diff
@@ -172,6 +173,13 @@ def set_expected_number(relativity, count)
                                end
           end
 
+          def serialize_and_deserialize_arguments(args)
+            serialized = ::ActiveJob::Arguments.serialize(args)
```

Comment:
I am surprise we have to serialize then deserialize. I am wondering if it can be simpler?

https://github.com/rails/rails/blob/6-0-stable/activejob/lib/active_job/serializers/time_serializer.rb

Reply:
I changed to use simple time rounder instead of serializer/deserializer.
It is copied from ActiveJob::TestHelper.

https://github.com/rails/rails/blob/c986369095ad89fd68fd84d21aa39006ea77904d/activejob/lib/active_job/test_helper.rb#L638

Reply:
I prefer the original implementation, it's simpler as it replicates what Rails does without special casing, the new implementation may not catch other edgecases and we're just reinventing what Rails is doing.

Reply:
I'd also like to see a green build, which is why I started #2158

Reply:
I also like the original implementation, so I reverted. And, I deleted `ActiveJob::DeserializationError` because it never occurred.

Reply:
Thanks for the change and sorry for the back and forth.